app/controllers/frontend_controller.rb

```diff
@@ -108,6 +109,10 @@ def pick_buyer_or_provider_layout
     end
   end
 
+  def all_services
+    @api_selector_services = current_user.accessible_services.to_a if current_account.try!(:provider?)
```

Comment:
IMO it should be just `@api_selector_services = current_user.accessible_services.to_a if current_account.provider?` because at the beginning of this same controller there is a `before_action :login_required` that verifies that the `current_account` exists 😄 
No need to make a new commit only for this, I am telling you just in case you already need to push for anything else, otherwise just ignore it 👍

Reply:
fair enough, will add to list for next iteration.

Reply:
Done in https://github.com/3scale/porta/pull/118